child_compassion/__openerp__.py

```diff
@@ -56,7 +56,11 @@
         'data/delegate_child_compassion_cron.xml',
         'data/child_compassion_template.xml',
     ],
-    'demo': [],
+    'demo': [
+        'demo/compassion.country.csv',

```

Comment:
This file is not needed. When creating children, the project will be automatically created with its country.